README.md

```diff
@@ -8,6 +8,12 @@
 
     python3 -m pip install eth_utils web3
 
+To run the backend on **Ubuntu >= 24.04**:
```

Comment:
We can merge this to the pre-requisites section above since they're all Ubuntu/debian based instructions.

In order for `liboxenc-dev` and `liboxenmq-dev` to be installed successfully the instructions for configuring the `Oxen Deb Repository` needs to be pulled up above so that it's done first before proceeding with the install. Which is good we can remove the section below and just have one `apt install` instruction that does it all in one.

Then the python command should be updated to invoke pip module through python, `python3 -m pip install <deps>` which makes sure that the pip you use installs the files for the `python` instance available on the path.

I'd also remove `python3.12-venv` as we don't use virtual environments in the instructions.